process_manager/tables.py

```diff
@@ -43,6 +54,23 @@ class Meta:  # noqa: D106
     select = tables.CheckBoxColumn(
         accessor="uuid",
         verbose_name="Select",
-        checked="checked",
-        attrs={"th__input": {"onclick": "toggle(this)"}},
+        attrs={
+            "th__input": {
+                "id": "header-checkbox",
+                "hx-preserve": "true",
+                "_": header_checkbox_hyperscript,
+            }
+        },
     )
+
+    def render_select(self, value: str) -> str:
```

Comment:
What's `value` here? Is the `uuid` of the column? Whatever it is, maybe add it to the docstring.

Also, I'm not sure when this method is called, or by whom.

Reply:
Thanks good catch. I wish there was a way to get automatic checks for this but I've never been able to find one.

It's called during table rendering. It's detected by class magicness and called in place of the render method of the column.

Reply:
`mkdocs` gives warnings about malformed docstrings (=errors if you pass `--strict`), but I don't know if it warns about missing args. Were we going to add API documentation at some point?

Reply:
Planning to add it for next sprint.